spec/controllers/calculator_constants_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/controller_macros'
+require 'support/devise'
+require 'controllers/shared_examples/shared_examples_for_authentication'
+
+RSpec.describe CalculatorConstantsController, type: :controller do
```

Comment:
Need to add something like below to use the `shared_examples_for_authentication`
```
  it_behaves_like 'an authenticating controller', :partials, 'calculator_constants'
```

Reply:
Added it in.